content/en/docs/cloud-api-adaptor/azure/_index.md

```diff
@@ -222,27 +182,151 @@ export AZURE_SUBNET_ID=$(az network vnet subnet list \
 ```
 
-> **Note**:
->
-> - If you want to use the nightly build of pod VM image then follow the instructions [here](azure-advanced#use-nightly-builds).
-> - If you want to build the pod VM image yourself then follow the instructions [here](azure-advanced#build-a-custom-pod-vm-image).
+{{% /tab %}}
+
+{{% tab header="Latest Build" %}}
+
+An automated job builds the pod VM image each night at 00:00 UTC. You can use that image by exporting the following environment variable:
+
+```bash
+SUCCESS_TIME=$(curl -s \
+  -H "Accept: application/vnd.github+json" \
+  "https://api.github.com/repos/confidential-containers/cloud-api-adaptor/actions/workflows/azure-podvm-image-nightly-build.yml/runs?status=success" \
+  | jq -r '.workflow_runs[0].updated_at')
+
+export AZURE_IMAGE_ID="/CommunityGalleries/cocopodvm-d0e4f35f-5530-4b9c-8596-112487cdea85/Images/podvm_image0/Versions/$(date -u -jf "%Y-%m-%dT%H:%M:%SZ" "$SUCCESS_TIME" "+%Y.%m.%d" 2>/dev/null || date -d "$SUCCESS_TIME" +%Y.%m.%d)"
+```
+
+Above image version is in the format `YYYY.MM.DD`, so to use the latest image should be today's date or yesterday's date.
+
+{{% /tab %}}
+
+{{% tab header="DIY" %}}
+
+If you have made changes to the CAA code that affects the pod VM image and you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/azure/build-image.md) to build the pod VM image. Once image build is finished then export image id to the environment variable `AZURE_IMAGE_ID`.
+
+{{% /tab %}}
+
+{{< /tabpane >}}
 
 ### CAA container image
 
+{{< tabpane text=true right=true persist=header >}}
+{{% tab header="**Versions**:" disabled=true /%}}
+
+{{% tab header="Last Release" %}}
+
+Export the following environment variable to use the latest release image of CAA:
+
 ```bash
 export CAA_IMAGE="quay.io/confidential-containers/cloud-api-adaptor"
 export CAA_TAG="d4496d008b65c979a4d24767979a77ed1ba21e76"
 ```
 
-> **Note**: If you have made changes to the CAA code and you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/install/README.md#building-custom-cloud-api-adaptor-image) to build the container image. Once the image is built export the environment variables accordingly.
+{{% /tab %}}
+
+{{% tab header="Latest Build" %}}
+
+Export the following environment variable to use the image built by the CAA CI on each merge to main:
+
+```bash
+export CAA_IMAGE="quay.io/confidential-containers/cloud-api-adaptor"
+```
+
+Find an appropriate tag of pre-built image suitable to your needs [here](https://quay.io/repository/confidential-containers/cloud-api-adaptor?tab=tags&tag=latest).
+
+```bash
+export CAA_TAG=""
+```
+
+> **Caution**: You can also use the `latest` tag but it is **not** recommended.
```

Comment:
Can we add a statement why that's a bad idea? (apart from using latest being a bad practice)

Reply:
Sure.

Reply:
Done